cli/standalone/virtual_fs.rs

```diff
@@ -174,6 +174,20 @@ impl VfsBuilder {
   pub fn into_dir_and_files(self) -> (VirtualDirectory, Vec<Vec<u8>>) {
     (self.root_dir, self.files)
   }
+
+  fn path_relative_root(&self, path: &Path) -> PathBuf {
+    match path.strip_prefix(&self.root_path) {
+      Ok(p) => p.to_path_buf(),
+      Err(err) => {
+        panic!(
+          "Failed to strip prefix '{}' from '{}': {:#}",
+          self.root_path.display(),
+          path.display(),
+          err
+        )
+      }
```

Comment:
Do we need to panic here? If the stripping fails we could maybe continue with the existing path?

Reply:
I'd like to understand the issue first. I'm hoping to ship this in canary and then go from there.